src/client/components/DashbordItems/Dashbord.style.css

```diff
@@ -2,20 +2,18 @@
   padding: 0;
 }
 .list li {
-  border-radius: 10px;
+  border-radius:var( --border-radius-low);
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Comment:
you should remove this, because font-family should come from index.css and it is different font-family than the one you used
```suggestion
```